src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -584,14 +584,6 @@ static inline int MPIDI_NM_mpi_put(const void *origin_addr,
         goto fn_exit;
     }
 
-    MPIDI_CH4U_EPOCH_CHECK_SYNC(win, mpi_errno, goto fn_fail);
```

Comment:
I don't think we want to move these checks into the CH4 layer. It is up to the netmod to call these routines, as they are fallbacks. A netmod could also use its own sync routines, in which case there could be conflicts.